oidc-api/src/main/java/uk/gov/di/authentication/oidc/services/OrchestrationAuthorizationService.java

```diff
@@ -222,6 +222,8 @@ public void storeState(String sessionId, State state) {
     }
 
     public boolean isJarValidationRequired(ClientRegistry client) {
-        return client.getScopes().contains(CustomScopeValue.DOC_CHECKING_APP.getValue());
+        return client.getScopes().contains(CustomScopeValue.DOC_CHECKING_APP.getValue())
+                ? client.getScopes().contains(CustomScopeValue.DOC_CHECKING_APP.getValue())
+                : client.isJarValidationRequired();
```

Comment:
Is this different from
```suggestion
        return client.getScopes().contains(CustomScopeValue.DOC_CHECKING_APP.getValue()) || client.isJarValidationRequired();
```
?

Reply:
Good point, shall change that :)